src/components/Breadcrumbs.js

```diff
@@ -19,7 +19,7 @@ const Breadcrumbs = () => {
       {breadcrumbs.map(({ breadcrumb, key, location, match }) => {
         // `/pings` is included in the routes, but there is not a /pings page,
```

Comment:
Should this comment be updated to include `/stream`? Or does that not apply the same as with `/pings`?